CHANGELOG.md

```diff
@@ -1,6 +1,378 @@
+     - Refactor lockstep programming #3616
+     - Change ExchangeTypeToRank from protected to public #3718
+     - Clarify naming and comments for pmacc kernel mapping types #3765
+     - Separate notification of plugins into a new function in SimulationHelper #3788
+     - Add a factory and a factory function for pmacc::StrideMapping #3785
+     - Replace custom compile-time string creation by BOOST_METAPARSE_STRING #3792
+     - Refactor CachedBox #3813
+     - Refactor and extend Vector #3817
+     - Drop unused TwistedAxesNavigator #3821
+     - Refactor DataBox #3820
+     - Replace PitchedBox ctor with offset by DataBox.shift() #3828
+     - Rename type to Reference in Cursor #3827
+     - Fix a warning in MapperConcept #3777
+     - Remove leftover mentions of ADIOS1 #3795
+   - plugins:
+     - Remove unused ParticlePatches class #3419
+     - Switch to openPMD API in PhaseSpace plugin #3468
+     - Switch to openPMD API in MacroParticleCounter plugin #3570
+     - Switch to openPMD API in ParticleCalorimeter plugin #3560
+     - Vector field vis vis compatility and benchmarking in ISAAC plugin #3719
+     - Remove libSplash #3744
+     - Remove unnecessary parameter in writeField function template of openPMD plugin #3767
+     - Remove --openPMD.compression parameter #3764
+     - Rename instance interface for multiplugins #3822
+     - Use multidim access instead of two subscripts #3829
+     - Use Iteration::close() in openPMD plugin #3408
+   - tools:
+     - Add cmake option to enforce dependencies #3586
+     - Update to mallocMC 2.5.0crp-dev #3325
+     - Clarify output concerning cuda_memtest not being available #3345
+   - Reduce complexity of examples #3323
+   - Avoid species dependency in field solver compile test #3430
+   - Switch from Boost tests to Catch2 #3447
+   - Introduce clang format #3440
+   - Use ubuntu 20.04 with CUDA 11.2 in docker image #3502
+   - Apply sorting of includes according to clang-format #3605
+   - Remove leftover boost::shared_ptr #3800
+   - Modernize according to clang-tidy #3801
+   - Remove more old boost includes and replace with std:: counterparts #3819
+   - Remove boost::result_of #3830
+   - Remove macro __deleteArray()  #3839
+ - documentation:
+   - Fix a typo in the reference to [Pausch2018] #3214
+   - Remove old HZDR systems #3246
+   - Document C++14 requirement #3268
+   - Add reference in LCT Example #3297
+   - Fix minor issues with the sphinx rst formating #3290
+   - Fix the name of probe fieldE and fieldB attribute in docs #3385
+   - Improve clarity of comments concerning density calculation and profile interface #3414
+   - Clarify comments for generating momentums based on temperature #3423
+   - Replace travis badges with gitlab ones #3451
+   - Update supported CUDA versions #3458
+   - Juwels profile update #3359
+   - Extend reminder to load environment in the docs with instructions for spack #3478
+   - Update license headers for 2021 #3486
```

Comment:
IMO we can remove this it is definitely not importent for the user

Reply:
I think only the last one is truly unimportant. The JUWELS profile could in principle be reused (rather copied-and-adjusted), and the other two are extension to docs. So i would slightly prefer keep the first 3